flags/flags.go

```diff
@@ -6,13 +6,11 @@ import (
 	"path/filepath"
 
 	goFlags "github.com/jessevdk/go-flags"
-	"github.com/wtfutil/wtf/help"
 	"github.com/wtfutil/wtf/utils"
 )
 
 type Flags struct {
 	Config  string `short:"c" long:"config" optional:"yes" description:"Path to config file"`
-	Module  string `short:"m" long:"module" optional:"yes" description:"Display info about a specific module, i.e.: 'wtf -m=todo'"`
```

Comment:
I ripped this out as it doesn't seem as useful/it is not getting updated with new modules.

Reply:
This allows someone to view the help text of a module from the command line without having to create a config for the module and run the app, eg: `wtf -m=todo` will display the help text for the Todo module. As long as new modules have help text in them, this works.

I don't want to lose this. Help text as it is now is the most basic implementation possible, it could be much more than just "this key does that" - think `man` pages for WTF modules, including configuration instructions, API links, etc. I just haven't spent any time on that side of things yet.